SimpleMap/Views/ContentView.swift

```diff
@@ -9,24 +9,45 @@ import SwiftUI
 import MapKit
 
 struct ContentView: View {
-	@State var Locations: [Location] = []
-	@State private var region = MKCoordinateRegion(center: CLLocationCoordinate2D(latitude: 51.507222, longitude: -0.1275), span: MKCoordinateSpan(latitudeDelta: 10, longitudeDelta: 10))
+    
+    @State var Locations: [Location] = []
+    @State var pinClicked: Bool = false
+    @State private var region = MKCoordinateRegion(center: CLLocationCoordinate2D(latitude: 51.507222, longitude: -0.1275), span: MKCoordinateSpan(latitudeDelta: 10, longitudeDelta: 10))
```

Comment:
instead of hardcoding lat/lon think about how you could use CoreLocation to fetch the mobile device's gps coordinates. For context see https://www.hackingwithswift.com/read/22/2/requesting-location-core-location